test/lsp-completion-test.el

```diff
@@ -73,8 +73,8 @@
              '("F" "daFo" "safo")
              '("F" "daFo" "safo"))
     (do-test "foo"
```

Comment:
can you add a single letter prefix test like:

> f foo Foo aFoo afoo

Reply:
and maybe 
> F foo Foo aFoo afoo

Reply:
Done